content/tutorials/react-setup.md

```diff
@@ -0,0 +1,102 @@
+---
+title: React App Setup
+description: Tutorial to set up a basic React app that uses squid-js.
+---
+
+This tutorial shows how you can build a basic [React](https://reactjs.org/) app that uses the squid-js JavaScript package to publish a data set, get a data set, and more.
```

Comment:
```suggestion
This tutorial shows how you can build a basic [React](https://reactjs.org/) app with [Create React App](https://github.com/facebook/create-react-app) that uses the squid-js JavaScript package to publish a data set, get a data set, and more.
```

Also, that intro probably should be the `description` value in the frontmatter above.

Reply:
@jernejpregelj Because you created this pull request, only you can accept these "Suggested changes" (via the GitHub user interface).